openfl/component/aggregator/aggregator.py

```diff
@@ -186,6 +187,8 @@ def __init__(
         # Initialize a lock for thread safety
         self.lock = Lock()
 
+        self.delta_updates = delta_updates
+
```

Comment:
Boolean state variables (and function arguments) should be named as `self.use_delta_updates` - improves readability.